normalizer/normalizer.go

```diff
@@ -16,31 +16,38 @@ const (
 	output  string = "normalized"
 )
 
+// Service represents Normalizer service.
+type Service interface {
```

Comment:
This is unnecessary, especially having in mind that it exposes a callback. I would prefer that you remove it and keep it as it is for now.